bootcode/bootcode.js

```diff
@@ -0,0 +1,26 @@
+(function (g) {
```

Comment:
Suggest @Leechael to review this part. It injects some objects to the js global scope.

Reply:
I suggest better using named export here.

Consider:

```javascript
import { pink } from "@kvinwang/pink-env";
```

Avoid namespace conflict:

```javascript
import { pink as PhalaPink } from '@kvinwang/pink-env';
```

Also, if we do `import "@kvinwang/pink-env"` in a few scripts and combine that all together, it might did the global injection multiple times.

Reply:
This is the boot code that will run to initialize the JS engine.
Do you mean that we should turn this piece of code into a js module? Is there a way in pure JS to define a global module that can be imported later in the user code?
@Leechael

Reply:
Yes, avoid injecting any global variables/functions into users' environment directly without explicit declaration.

Reply:
As discussed in the discord. We currently haven't a better way. Let's just keep it as is.